amazon-speech-api/amazonTranscribeHarris.js

```diff
@@ -35,40 +118,100 @@ function startTranscriptionJob(transcriptionObject) {
     OutputBucketName: transcriptionObject.outputBucketName,
     TranscriptionJobName: transcriptionObject.jobName, /* required */
     Settings: {
-      // ChannelIdentification: false,
-      // MaxSpeakerLabels: 0,
-      // ShowSpeakerLabels: false
+      VocabularyName: vocabularyObject.VocabularyName // To set the vocabulary
```

Comment:
have you noticed any positive or negative difference on the transcription quality now that the vocabulary is being set?